src/panels/lovelace/components/hui-warning.ts

```diff
@@ -0,0 +1,42 @@
+import {
+  html,
+  LitElement,
+  TemplateResult,
+  property,
+  CSSResult,
+  css,
+} from "lit-element";
+
+export class HuiWarning extends LitElement {
+  @property() public warning?: string;
+  @property() public entity?: string;
+
+  public getCardSize(): number {
+    return 1;
+  }
+
+  protected render(): TemplateResult | void {
+    return html`
+      ${this.warning || "Entity not available"}: ${this.entity || ""}
+    `;
+  }
+
+  static get styles(): CSSResult {
+    return css`
+      :host {
+        display: block;
+        color: black;
+        background-color: yellow;
+        padding: 8px;
+      }
+    `;
+  }
+}
+
+declare global {
+  interface HTMLElementTagNameMap {
+    "hui-warning": HuiWarning;
+  }
+}
+
+customElements.define("hui-warning", HuiWarning);
```

Comment:
Use the decorator `@customElement("hui-warning")` and add on new line above the class definition

```
@customElement("hui-warning")
class HuiWarning extends LitElement {
```

Reply:
decorators are the best!

Reply:
I'll probably do a PR today or tomorrow to do this across the board on LitElements, along with property and static css.